prompt.js

```diff
@@ -32,8 +35,17 @@ module.exports = {
       pageSize: 2,
       choices: [
         { message: "Axios", name: "axios" },
-        { message: "Prettier", name: "prettier" },
-        { message: "Jest", name: "jest" },
+        { message: "Prettier", name: "prettier", disabled: true },
+        { message: "ESLint", name: "eslint", disabled: true },
+        { message: "Reverse Proxy", name: "reverse-proxy", disabled: true },
+        { message: "Bundle Analyzer", name: "bundle-analyzer", disabled: true },
+        { message: "i18n", name: "i18n", disabled: true },
+        { message: "Storybook", name: "storybook", disabled: true },
+        { message: "dotenv", name: "dotenv", disabled: true },
```

Comment:
```suggestion
        { message: "dotenv", name: "dotenv", disabled: true },
```
dotenv değil de, next'in built-in gelen env'si olmalı